src/http/Request.php

```diff
@@ -127,10 +127,17 @@ class Request implements RequestInterface {
   /**
    * Holds the parsed body.
    *
+   * @var array
```

Comment:
That file is not in my local :-(

I tried removing it and re-creating it, but it did not help…